src/views/Peers.tsx

```diff
@@ -71,22 +75,28 @@ export const Peers = () => {
     const actionsMenu = (<Menu items={itemsMenuAction} ></Menu>)
 
     const transformDataTable = (d:Peer[]):PeerDataTable[] => {
-        return d.map(p => ({ key: p.IP, ...p } as PeerDataTable))
+        const peer_ids = d.map(_p => _p.ID)
+        return d.map((p) => {
```

Comment:
Codacy has a fix for the issue: [Replace `········` with `····`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
    return d.map((p) => {
```